File: ontrack-web-core/components/extension/jira/JiraServiceDeskRequestStatus.js
export const jiraServiceDeskRequestStatuses = [
    {
        value: 'ALL',
        label: "All requests",
    },
    {
        value: 'OPEN',
        label: "Only open requests",
    },
    {
        value: 'CLOSED',
        label: "Only closed requests",
    },
]

export default function JiraServiceDeskRequestStatus({status}) {
    return (
        <>
            {
                jiraServiceDeskRequestStatuses.find(it => it.value === status)?.label
            }
        </>
    )
}

File: ontrack-extension-jira/src/main/java/net/nemerosa/ontrack/extension/jira/servicedesk/JiraServiceDeskRequestStatus.kt
package net.nemerosa.ontrack.extension.jira.servicedesk

/**
 * Which requests to look for when looking for an existing ticket.
 *
 * See https://docs.atlassian.com/jira-servicedesk/REST/5.15.1/#servicedeskapi/request-getMyCustomerRequests
 */
enum class JiraServiceDeskRequestStatus(
    val requestStatus: String,
) {

    CLOSED("CLOSED_REQUESTS"),
    OPEN("OPEN_REQUESTS"),
    ALL("ALL_REQUESTS")

}